examples/repaire/main.go

```diff
@@ -13,8 +13,8 @@ import (
 //Пример чтения и сохранения нескольких LAS файлов
 func main() {
 	fileList := make([]string, 0, 10)
-	n := findFilesExt(&fileList, ".\\", ".las")
-	fmt.Printf("files founded: %d\n", n)
+	n := findFilesExt(&fileList, ".", ".las")
+	fmt.Printf("files found: %d\n", n)
```

Comment:
The word 'found' is one of the exceptions to adding 'ed' to a word to make it past-tense. 'found' is the past-tense of 'find'.